lib/puppet/network/formats.rb

```diff
@@ -151,7 +151,7 @@ def render(datum)
     end
 
     # ...or pretty-print the inspect outcome.
-    return json.render(datum)
+    return JSON.pretty_generate(datum)
```

Comment:
Yeah, the first version of this just used `PSON.pretty_generate`. I think it makes sense to deal with this patch and the PSON -> JSON transition as two separate things.

But short-circuit evaluation of things not responding to `to_json` seems like a reasonable workaround otherwise. This renderer already has a bunch of short circuit renders, so wouldn't really hurt to add a few more.